File: server/src/main/java/com/d201/goodshot/swing/dto/Similarity.java
package com.d201.goodshot.swing.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Similarity {

    private double address;
    private double finish;
    private double impact;
    private double midBackSwing;
    private double midDownSwing;
    private double midFollowThrough;
    private double toeUp;
    private double top;

}

File: android/domain/src/main/java/com/ijonsabae/domain/model/Similarity.kt
package com.ijonsabae.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class Similarity(
    val address: Double,
    val toeUp: Double,
    val midBackSwing: Double,
    val top: Double,
    val midDownSwing: Double,
    val impact: Double,
    val midFollowThrough: Double,
    val finish: Double
)
